### ISSUE TITLE
can't import hspy

### ISSUE THREAD

**Issue by biswajitSM** [tjof2/pgure-svt#56]
Thanks for the module. Looks interesting and I was trying to test it in an Ubuntu PC. I ran the example notebook to see if things are working, but got the following error:
![image](https://user-images.githubusercontent.com/12598051/90156731-17582000-dd8d-11ea-929b-1d7e63ed17ff.png)

Do you have any idea why isn't it working?

Thanks, Biswajit

**Comment by tjof2** [comment]
Does `from pguresvt.hspy import HSPYSVT` work instead? 

Then replace `hspy.HSPYSVT(...)` with `HSPYSVT(...)`

**Comment by biswajitSM** [comment]
Thanks for the suggestion. It still doesn't work. The installation through the conda channel might not have worked properly.

**Comment by tjof2** [comment]
@ericpre I wonder if the commits after the last conda release might mean another one is needed?

I'm afraid I'm away from a PC for a few days @biswajitSM so it'll take a little time to fix it.

In the interim, you can try it without HyperSpy?

```python

import numpy as np
from pguresvt import SVT

# let X be your data as a 3d numpy array,
# shape (x pixels, y pixels, T frames)
s = SVT()
s.denoise(X)
s.Y_  # this is the denoised array
```

**Comment by ericpre** [comment]
I can reproduce the issue, to import the hyperspy wrapper, with the current release, you need to do:
```python
from pguresvt.hspysvt import HSPYSVT
```

The documentation is possibly out of date but it would be good to be able to:
```python
from pguresvt import SVT, HSPYSVT
```

Or even more simpler for the user, to be able to use SVT on numpy array or hyperspy signal!

**Comment by tjof2** [comment]
Indeed. I'll tidy up the docs and fix that when I'm back at a PC.

Thanks @ericpre  - your solution should work for @biswajitSM for now

**Comment by tjof2** [comment]
This is resolved in v0.6.2 such that the code and docs are in sync
